### PR TITLE
Remove python2 support

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

This pull request removes python2 support.
I have already checked issues:
- make rpm
- make test-in-container
- make srpm

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,7 +3,6 @@ sudo: false
 notifications:
```
Comment (TomasTomecek) [inline .travis.yml]: can you please remove the whole travis.yml, we don't use it any more

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ if the problem has already been reported (or fixed in a master branch).
 
 If you're unable to find an open issue addressing the problem, [open a new one](https://github.com/user-cont/conu/issues/new).
 Be sure to include a **descriptive title and a clear description**. Ideally, please provide:
- * version of conu you are using (`rpm -q python2-conu python3-conu` or `pip freeze | grep conu`)
+ * version of conu you are using (`rpm -q python3-conu` or `pip freeze | grep conu`)
```
Comment (TomasTomecek) [inline CONTRIBUTING.md]:
```suggestion
 * version of conu you are using (`rpm -q python3-conu` or `python3 -m pip freeze | grep conu`)
```

File: Makefile [hunk 3]
```diff
@@ -103,14 +102,6 @@ rpm-in-mock-f29: srpm
 rpm-in-mock-rawhide: srpm
 	mock --rebuild -r fedora-rawhide-x86_64 ./*.src.rpm
 
-rpm-in-mock-el7: srpm
```
Comment (TomasTomecek) [inline Makefile]: @user-cont/the-team is everyone okay with this? that we will drop conu from centos 7? do we actually have some centos users?

File: README.md [hunk 4]
```diff
@@ -11,6 +11,8 @@ and is handy when playing with containers inside your code.
 It defines an API to access and manipulate containers,
 images and provides more, very helpful functions.
 
+`conu` does NOT support Python 2 at all.
```
Comment (TomasTomecek) [inline README.md]: this sounds a little harsh

Comment (TomasTomecek) [inline README.md]:
```suggestion
`conu` is supported on python 3.6+ only.
```

File: setup.py [hunk 5]
```diff
@@ -36,16 +36,7 @@ def get_requirements():
         for line in fd.readlines():
             if not line or line.startswith('-i'):
                 continue
-            parsed_line = line.split(';', 1)[0].strip()
```
Comment (TomasTomecek) [inline setup.py]: nice! we can reintroduce sesheta again

File: Makefile (hunk 3 above)
Comment (phracek) [reply Makefile]: I guess, python2 is mandatory on centos7 otherwise, we have to python3 from SCL org team, isn't it?

Comment (rpitonak) [reply Makefile]: There is still an option to install it from PyPI. +1 for removing this

**Comment by TomasTomecek** [comment]
I rescheduled the CI build:
```
requests.exceptions.ChunkedEncodingError: ("Connection broken: ConnectionResetError(104, 'Connection reset by peer')", ConnectionResetError(104, 'Connection reset by peer'))
```

seems like a flake

**Comment by phracek** [comment]
All tests passed but the script fails on `source-to-image`

**Comment by TomasTomecek** [comment]
FML, how could the CI pass when we changed the spec back then -_-
```
error: line 42: Unknown tag: Recommends:     source-to-image
```
So we need to do `make srpm` in a Fedora container. I vote to remove `srpm` target from the ci target and let packit handle this in future.

**Comment by phracek** [comment]
I vote as well.

**Pull request merged**